spec/call_flow_spec.rb

```diff
@@ -0,0 +1,60 @@
+describe "CallFlow" do
```

Comment:
please format the style of test according http://www.betterspecs.org/ 

at least, define variables using `let`